### PR TITLE
ROE-786 Add filing details for e-Mails

### PR DESCRIPTION
**Pull request by mwestacott**
* Filing description now set when getting filing details
* Unit test changes to check generated description

### CODE REVIEW

File: src/main/java/uk/gov/companieshouse/overseasentitiesapi/service/FilingsService.java
Comment (mouhajer-ch) [inline src/main/java/uk/gov/companieshouse/overseasentitiesapi/service/FilingsService.java:33]: Hi, do we need to add the following `OVERSEAS_ENTITIES_FILING_DESCRIPTION_IDENTIFIER` to our configs?

Comment (mwestacott) [reply src/main/java/uk/gov/companieshouse/overseasentitiesapi/service/FilingsService.java:33]: Yes. PRs to follow to do this (in Docker and chs-configs). Feel free to approve this PR but I won't merge until all PRs are ready to go.

**Review by mouhajer-ch — APPROVED** [review]

**Review by markpit — APPROVED** [review]

**Comment by mwestacott** [comment]
Will merge when config PRs are done and approved.

**Pull request merged**
